distribution/lib/Standard/Base/0.0.0-dev/src/Runtime.enso

```diff
@@ -0,0 +1,98 @@
+import Standard.Base.Data.Vector
+import Standard.Base.Polyglot
+import Standard.Base.Nothing
+from Standard.Base.Runtime.Extensions import Source_Location
+
+## Utilities for interacting with the runtime.
+##type Runtime
+##
+##    ## A container type for utility methods that allow interacting with the Enso
+##       runtime.
+##    @Builtin_Type
+##    type Runtime
```

Comment:
Remove stale commented out code?

Reply:
Likely related to the `IO`/`Prim_Io` questions - if this one is allowed to be removed, can we also remove these two?

Reply:
Yeap, leftover